homeassistant/components/dsmr/sensor.py

```diff
@@ -908,7 +928,17 @@ def __init__(
         self._attr_device_class = device_class
         self._attr_native_unit_of_measurement = native_unit_of_measurement
         self._entry = entry
-        self.telegram: Telegram | None = telegram
+        self._telegram: Telegram | None = telegram  # Stores the latest telegram
```

Comment:
Still added comment. Also for now avoid renaming with _ prefix so diff is easier to read.